packages/elements/examples/nextjs/app/sign-up/[[...sign-up]]/page.tsx

```diff
@@ -21,7 +21,70 @@ function CustomSubmit({ children }: ComponentProps<'button'>) {
 
 export default function SignUpPage() {
   return (
-    <SignUp>
+    <SignUp
```

Comment:
Is there potential of the start step acting as the fallback?

Reply:
We could, in practice there are some edge cases though, like if we're landing on a callback route. So I think the current guidance would be to abstract start and re-use it as your fallback if this is the desired behavior.